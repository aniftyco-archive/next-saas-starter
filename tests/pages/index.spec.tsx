import React from 'react';
import { render } from '@testing-library/react';
import Index from '../../src/pages/index';

test('renders app', () => {
  const { asFragment } = render(<Index />, {});
  expect(asFragment()).toMatchSnapshot();
});
