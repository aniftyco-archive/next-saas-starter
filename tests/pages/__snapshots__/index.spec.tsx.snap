// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col items-center justify-center h-screen"
  >
    <h2
      class="p-3 my-8 text-lg text-white font-bold bg-purple-700 md:text-2xl rounded-lg"
    >
      Hi! Welcome to your first Next.js & Tailwind CSS site.
    </h2>
  </div>
</DocumentFragment>
`;
